content/docs/1.6.0/references/settings.md

```diff
@@ -7,6 +7,7 @@ weight: 1
 - [General](#general)
   - [Node Drain Policy](#node-drain-policy)
   - [Automatically Cleanup System Generated Snapshot](#automatically-cleanup-system-generated-snapshot)
+  - [Automatically Cleanup Outdated Snapshots of Recurring Backup Jobs](#automatically-cleanup-outdated-snapshots-of-recurring-backup-jobs)
```

Comment:
"cleanup" is the noun form of the verb "clean up". I used "Clean Up" in my draft because it is the correct form to use in this title. If you want to use "Cleanup", we have to make a small change to the wording.

Two options:
1 - Automatically Clean Up Outdated Snapshots of Recurring Backup Jobs 
2 - Automatic Cleanup of Outdated Snapshots of Recurring Backup Jobs

I prefer option 1 because it reads better.

Reply:
Modified. Thanks.